cluster/evaluator/launch_eval.py

```diff
@@ -0,0 +1,119 @@
+import sys
+sys.path.insert(0, '.')
+
+import jinja2
+import kubernetes
+import yaml
+import json
+import os
+import argh
+import rl_loop
+import random
+import time
+
+
+def launch_eval(black_num=0, white_num=0):
+    if black_num <= 0 or white_num <= 0:
+        print("Need real model numbers")
+        return
+
+    b = rl_loop.get_model(black_num)
+    w = rl_loop.get_model(white_num)
+
+    b_model_path = os.path.join(rl_loop.MODELS_DIR, b)
+    w_model_path = os.path.join(rl_loop.MODELS_DIR, w)
+
+    kubernetes.config.load_kube_config()
```

Comment:
As mentioned in chat, this + the api instance should be called more regularly. load_kube_config refreshes GCP tokens; then, these get cached inside the configuration object.